source/run-command/tests/unified/runCursorCommand.yml

```diff
@@ -0,0 +1,488 @@
+description: runCursorCommand
+
+schemaVersion: '1.9'
+
+createEntities:
+  - client:
+      id: &failPointClient failPointClient
+      useMultipleMongoses: false
+  - client:
+      id: &commandClient commandClient
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent, commandSucceededEvent]
+  - client:
+      id: &client client
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent]
+  - session:
+      id: &session session
+      client: *client
+  - database: # For tests that need success event assertions
+      id: &commandDb commandDb
+      client: *commandClient
+      databaseName: *commandDb
+  - database:
+      id: &db db
+      client: *client
+      databaseName: *db
+  - collection:
+      id: &collection collection
+      database: *db
+      collectionName: *collection
+
+initialData:
+  - collectionName: collection
+    databaseName: *db
+    documents: &documents
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+      - { _id: 4, x: 44 }
+      - { _id: 5, x: 55 }
+
+tests:
+  # This is what this API was invented to do.
+  - description: successfully executes checkMetadataConsistency cursor creating command
+    runOnRequirements:
+      - minServerVersion: '7.0'
+        topologies: [sharded]
+    operations:
+      - name: runCursorCommand
+        object: *db
+        arguments:
+          commandName: checkMetadataConsistency
+          command: { checkMetadataConsistency: 1 }
+        expectResult: []
```

Comment:
Quoting the runCommand spec:

> If the response from the server does not include a cursor field the driver MUST throw an error either before returning from runCursorCommand or upon first iteration of the cursor.

Also, quoting the language you added for `runCursorCommand` operation in the Unified Test Format spec:

> When executing the provided command, the test runner MUST fully iterate the cursor.
This will ensure consistent behavior between drivers that eagerly create a server-side cursor and those that do so lazily when iteration begins.

If you remove `expectResult` here, the test runner is still going to ensure that the command was successful and that the cursor was iterated without error. We just won't bother asserting the contents of that cursor.

I think we can safely remove it.

If you're not confident in the test, you could consider splitting this off to a separate file and utilizing a `CommandSucceededEvent` to assert that the response includes a `cursor` field.

Reply:
Removed the expect result, I'm confident in the test proving that we can make a cursor out of this new command